packages/core/EventManager.ts

```diff
@@ -545,6 +561,14 @@ export default class EventManager {
           );
           // It might not be the first connected calendar as it seems that the order is not guaranteed to be ascending of credentialId.
           const firstCalendarCredential = destinationCalendarCredentials[0];
+
+          if (!firstCalendarCredential) {
+            log.warn(
+              "No other credentials found of the same type as the destination calendar. Falling back to first connected calendar"
+            );
+            await fallbackToFirstConnectedCalendar();
+          }
```

Comment:
If there is no second credential of the same type as the destination calendar, then fallback to the first connected calendar.